src/pages/trails/ballots/view/BallotView.vue

```diff
@@ -218,35 +226,26 @@ export default {
         let votes = this.userVotes[ballot_name].weighted_votes.map(v => v.key);
         this.votes = this.votes.concat(votes);
     },
-    // ---- quickfix for #92 -------
-    ...mapActions('trails', ['registerVoter']),
-    async onRegisterVoter (max_supply) {
-      await this.registerVoter(max_supply);
-    },
-    // -------------------------------
     async vote() {
         let register = false;
-        if (this.isUserRegisteredInTreasury) {
-            register = false;
-        } else {
-            // ---- quickfix for #92 -------
-            /*
-            if (this.ballot.treasury.access == 'public') {
-                register = true;
+
+        if (this.isPositiveVotePower) {
```

Comment:
Treasuries that consider the liquid balance of the accounts can create ballots that disregard the staked balance for voting. [This issue](https://github.com/telosnetwork/app-telos-net/issues/44) was created to address the `votestake` and `voteliquid` behavior a while back.

I have created [this beauty](https://miro.com/app/board/uXjVPQ6EDvA=/) to map the flow. The opening tutorial in another tab is my suggestion for guiding the user to the next step.

Reply:
@karynemayer, I just sent an Access request for the Miro site (using my normal email viter.rod@gmail.com)

Reply:
@Viterbo I've updated the link and granted you editor access.

Reply:
In this commit [25ceabb](https://github.com/telosnetwork/app-telos-native/pull/171/commits/25ceabb7db186b24201787a8ce06943d023dc828), the official ballot cases (with `VOTE` symbol) are handled separately from those that are not. 

In the latter case (with custom symbol), it is also handled properly if the ballot is `votestake` and when it is `voteliquid`.